traitsui/qt4/toolkit.py

```diff
@@ -26,10 +26,17 @@
 _app = pyface_toolkit("init:_app")
 
 from traits.trait_notifiers import set_ui_handler
+from pyface.api import SystemMetrics
 from pyface.qt import QtCore, QtGui, qt_api
 
 from traitsui.toolkit import Toolkit
-from .constants import screen_dx, screen_dy
+
+
+#: Screen width
+screen_dx = SystemMetrics()._get_screen_width()
+
+#: Screen height
+screen_dy = SystemMetrics()._get_screen_height()
```

Comment:
I believe these new definitions are in response to this [comment](https://github.com/enthought/traitsui/issues/1316#issuecomment-704903499):

> A stretch goal would be to remove these values from constants.py and just use the Pyface class directly wherever it is needed (which I am guessing is only a few places).

The current move does not change when the screen width/height are evaluated. The dimensions are still evaluated at import time. 
I think what Corran had meant was to get the screen dimensions where they are used, e.g here:
https://github.com/enthought/traitsui/blob/32458481d7a38f3d4811020fcf1837ea1b8088c2/traitsui/wx/helper.py#L231
Such move is more likely to cause behavioural change:  currently the screen size is computed when the code is loaded (e.g. during `import traitsui.api`), the move to inline code will cause the screen size to be computed when it is needed, which could be much later, like when a widget is painted. This may well be a good and a correct thing to do, but is one that requires a bit more thoughts and evaluations.

@corranwebster Did I understand your comment correctly?

Reply:
Ah I see.  I had started to do this initially, but second guessed if it was the right thing.  What you described makes sense to me so I am happy to make that change now. But we can wait for Corran's response too if you prefer.

Reply:
I am fairly sure that was what @corranwebster meant, blame it on me if I get it wrong then :)

Reply:
Yes, I was suggesting what Kit is suggesting above.  It's sort of orthogonal to the PR, but it will lead to slightly better behaviour if screen size changes.